docs/tutorials/write-a-comit-app.md

```diff
@@ -1,7 +1,59 @@
 ---
 id: write-a-comit-app
-title: Write your own COMIT App
-sidebar_label: Write-a-comit-app
+title: Write your first COMIT-app
+sidebar_label: Write your first COMIT-app
 ---
 
-It is not ready yet, come back soon!
\ No newline at end of file
+// TODO: Explain what this file is about and the goal of the tutorial (3-4 Sentences).
+
+This is a typescript tutorial for creating your first COMIT-app. 
+In this tutorial we build a simple command-line application that uses the COMIT protocol to exectute an atomic swap locally on your machine between Bitcoin and Ethereum.
+
+## The story
+
+// TODO Describe a simple scenario that introduces the roles and what they want to do.
+
+Alice still has some Ether lying around from the early crypto days.
+Recently she took a look at Ethereum 2.0 she could not believe her eyes. (...)
+Scared of the Ethereum developments she is thinking about moving some of her Ether into Bitcoin.
```

Comment:
I don't think this piece of opinion has its place here (or on any doc produced by CoBloX).